blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxReader.java

```diff
@@ -198,6 +197,12 @@ public static AstyanaxReader getInstance() {
             for (Column<Long> col : cols) {
                 points.add(new Points.Point<T>(col.getName(), (T)col.getValue(serializer)));
             }
+
+            // we only want to count the number of points we
+            // get when we're querying the metrics_full
+            if ( cf == CassandraModel.CF_METRICS_FULL || cf == CassandraModel.CF_METRICS_PREAGGREGATED_FULL ) {
+                Instrumentation.getRawPointsIn5MinHistogram().update(points.getPoints().size());
```

Comment:
I am thinking, we should only do this for regular metrics. Thats the one which is likely to have higher numbers.